docs/user/guides/set-up-single-sign-on-saml.rst

```diff
@@ -0,0 +1,125 @@
+   Contact :doc:`support </support>` to enable this feature for your organization.
+
+   **At the moment only Okta is supported as a SAML identity provider.**
+
+This how-to guide will provide instructions on how to enable :abbr:`SSO (Single Sign-on)` using Okta as a SAML identity provider.
+If you want more information on this feature, please read :doc:`/commercial/single-sign-on`
+
+Prerequisites
+-------------
+
+.. include:: /shared/organization-permissions.rst
+
+Create a new SAML application in Okta
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to enable SSO with Okta, you need to create a new SAML application in your Okta account.
+
+1. Log in to your Okta account.
+2. Click on :guilabel:`Applications`.
+3. Click on :guilabel:`Create App Integration`.
+4. Choose :guilabel:`SAML 2.0`, and click :guilabel:`Next`.
+5. Fill in the following fields with the information from `your SAML integration <https://readthedocs.com/organizations/choose/organization_saml/>`__:
+
+   * :guilabel:`App name`: Read the Docs
+   * :guilabel:`App logo`: Optionally you can use the `Read the Docs logo <https://brand-guidelines.readthedocs.org/_images/logo-wordmark-vertical-compact-dark.png>`__.
+   * :guilabel:`App visibility`: (optional)
+   * :guilabel:`Single Sign On URL`: ``https://readthedocs.com/accounts/saml/<organization-slug>/acs/`` (replace ``<organization-slug>`` with your organization slug)
+   * :guilabel:`Audience URI (SP Entity ID)`: ``https://readthedocs.com/accounts/saml/<organization-slug>/metadata/`` (replace ``<organization-slug>`` with your organization slug)
+   * :guilabel:`Name ID format`: ``EmailAddress``
+   * Leave the rest of the fields as default.
+
+6. Add the following "attribute statements" to be used when creating a new user:
+
+   .. csv-table::
+      :header: "Name", "Format", "Value"
+
+      "user.id", "Basic", "user.id"
+      "user.firstName", "Basic", "user.firstName"
+      "user.lastName", "Basic", "user.lastName"
+
+7. Click :guilabel:`Next`.
+8. Select ``This is an internal app that we have created``.
+9. Click :guilabel:`Finish`.
+
+User setup
+~~~~~~~~~~
+
+Using this setup, all users who have access to the configured Okta application will automatically join to your organization when they sign up.
+Existing users will not be automatically joined to the organization.
+
+You can still add outside collaborators and manage their access.
+There are two ways to manage this access:
+
+* Using :doc:`teams </guides/manage-read-the-docs-teams>` to provide access for ongoing contribution.
+* Using :doc:`sharing </commercial/sharing>` to provide short-term access requiring a login.
+
+Enabling SSO
+------------
+
+Enabling SSO is currently done by the Read the Docs team,
+contact :doc:`support </support>` to enable this feature for your organization.
+
+By default, users that sign up with SAML do not have any permissions over any project.
+However, you can define which teams users will auto-join when they sign up.
```

Comment:
Aren't we creating a team automatically when SAML is enabled on an organization in a similar way as we are doing with Google SSO? If not, we should probably do the same and enable auto-join on that team. I think it's a good idea to keep consistency between these two SSO providers.

Reply:
A team is automatically created.